lib/command/run.rb

```diff
@@ -89,10 +91,12 @@ class Run < Base # rubocop:disable Metrics/ClassLength
 
     DEFAULT_JOB_CPU = "1"
     DEFAULT_JOB_MEMORY = "2Gi"
+    DEFAULT_JOB_TIMEOUT = 21_600 # 6 hours
+    DEFAULT_JOB_HISTORY_LIMIT = 10
```

Comment:
So we can see the maximum number of jobs in the Control Plane UI.

Reply:
👍